css/properties/grid-gap.json

```diff
@@ -0,0 +1,41 @@
+{
+  "css": {
+    "properties": {
+      "grid-gap": {
+        "__compat": {
+          "spec_url": "https://drafts.csswg.org/css-align-3/#gap-legacy",
+          "support": {
+            "chrome": {
+              "version_added": "≤80"
+            },
+            "chrome_android": "mirror",
+            "edge": {
+              "version_added": "≤80"
+            },
+            "firefox": {
+              "version_added": "≤72"
```

Comment:
Works for grid in Fx 52, and flex in Fx 63, but I don't know how granular (or how far back) you want to go with these.